client/app/profile/profile.html

```diff
@@ -262,6 +262,14 @@ <h4>{{ 'Set mapper level' | translate }}</h4>
                                     <p ng-show="profileCtrl.errorSetLevel" class="error">{{ 'An error occurred when setting the level.' | translate }}</p>
                                 </div>
                             </div>
+                            <div>
+                                <h4>{{ 'Expert Mode' | translate }}</h4>
+                                <p>{{ 'Expert mode makes advanced and experimental features available. Most users should leave this off.' | translate }}</p>
+                                <label class="form__option">
+                                    <input type="checkbox" ng-model="profileCtrl.userDetails.isExpert" ng-change="profileCtrl.setExpertMode(profileCtrl.userDetails.isExpert)"/>{{ 'Expert Mode' | translate }}
+                                </label>
+                                <p ng-show="profileCtrl.errorSetExpertMode" class="error">{{ 'An error occurred when altering expert mode setting.' | translate }}</p>
```

Comment:
Hi Neil, would it be possible to add some confirmation to users when it saves successfully? That is my only comment on the PR--otherwise everything looks good!

Reply:
Great idea, @ethan-nelson. Saving indicator has been added.

![expert_saving](https://user-images.githubusercontent.com/445970/46750892-e4fbaa00-cc6d-11e8-898a-75849d00a663.gif)